lib/util/exec/docker/index.spec.ts

```diff
@@ -0,0 +1,300 @@
+import { getName } from '../../../../test/util';
+import { setAdminConfig } from '../../../config/admin';
+import { SYSTEM_INSUFFICIENT_MEMORY } from '../../../constants/error-messages';
+import { getPkgReleases as _getPkgReleases } from '../../../datasource';
+import { logger } from '../../../logger';
+import { VolumeOption, rawExec as _rawExec } from '../common';
+import {
+  generateDockerCommand,
+  getDockerTag,
+  prefetchDockerImage,
+  removeDanglingContainers,
+  removeDockerContainer,
+  resetPrefetchedImages,
+} from '.';
+
+const rawExec: jest.Mock<typeof _rawExec> = _rawExec as any;
```

Comment:
Why not using `test/exec-util` as in other tests?

Reply:
Haha, I forgot we have it 😅